src/portable/synopsys/dwc2/dcd_dwc2.c

```diff
@@ -543,7 +543,7 @@ void dcd_edpt_close_all(uint8_t rhport) {
   dwc2_regs_t* dwc2 = DWC2_REG(rhport);
   uint8_t const ep_count = _dwc2_controller[rhport].ep_count;
 
-  _dcd_data.allocated_epin_count = 1;
+  _dcd_data.allocated_epin_count = 0;
```

Comment:
We need to `_dcd_data.allocated_epin_count = 0` here, as it will be increased after EP0 fifo allocation in `dfifo_device_init(rhport);`